### PR TITLE
add fallback icons to all templates & translation of script command 

### PR DESCRIPTION
**Pull request by ReuschelCGN**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/lib/poracleMessage/commands/script.js [hunk 1]
```diff
@@ -319,21 +319,21 @@ exports.run = async (client, msg, args, options) => {
 		}
 
 		if (!message.length) {
-			return await msg.reply('The script specified is empty')
+			return await msg.reply(client.translator.translate('The script specified is empty'))
```
Comment (jfberry) [inline src/lib/poracleMessage/commands/script.js:322]: translator.translate(

client.translator.translate gives default language, translator.translate is user language

Comment (ReuschelCGN) [reply src/lib/poracleMessage/commands/script.js]: is changed

**Review by ReuschelCGN — COMMENTED** [review]
translation is fixed

File: src/lib/poracleMessage/commands/script.js (hunk 1 above)
Comment (jfberry) [reply src/lib/poracleMessage/commands/script.js]: well, that one is -- but there are others :-)

Comment (ReuschelCGN) [reply src/lib/poracleMessage/commands/script.js]: ALL fixed now! Sorry by the way, try to be more accurate in the future....

**Pull request merged**